### ISSUE TITLE
Pillow API compability issue

### ISSUE THREAD

**Issue by evilpan** [cc-d/flask-simple-captcha#20]
Seems that flask-simple-captcha only accepts Pillow>=9.0.0,<=9.20. for Pillow==9.5.0, it throws:

```
[2023-11-13 10:01:13,244] ERROR in app: Exception on /login [GET]
Traceback (most recent call last):
  File "/data/lib/python3.10/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
  File "/data/lib/python3.10/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/data/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/data/lib/python3.10/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/pan/bigfish/bigfish/views.py", line 49, in login
    new_captcha_dict = captcha.create()
  File "/data/lib/python3.10/site-packages/flask_simple_captcha/captcha_generation.py", line 158, in create
    out = self.draw_lines(out)
  File "/data/lib/python3.10/site-packages/flask_simple_captcha/captcha_generation.py", line 96, in draw_lines
    draw.ellipse(
  File "/data/lib/python3.10/site-packages/PIL/ImageDraw.py", line 168, in ellipse
    self.draw.draw_ellipse(xy, ink, 0, width)
ValueError: x1 must be greater than or equal to x0
```

and for the latest Pillow release, it throws:

```
[2023-11-13 09:59:26,929] ERROR in app: Exception on /login [GET]
Traceback (most recent call last):
  File "/data/lib/python3.10/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
  File "/data/lib/python3.10/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/data/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/data/lib/python3.10/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/src/views.py", line 49, in login
    new_captcha_dict = captcha.create()
  File "/home/pan/.local/lib/python3.10/site-packages/flask_simple_captcha/captcha_generation.py", line 144, in create
    text_size = fnt.getsize(text)
AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
```

**Comment by evilpan** [comment]
In the README.md, it shows Pillow<=9.0.0, but in the `setup.py` it's `>=9.0.0`

```py
    install_requires=[
        'Werkzeug<3,>=0.16.0',
        'Pillow>=9.0.0',
        'myfuncs==1.4.8',
        'pyjwt<3,>=2',
    ],
```

So Installing this library via pip would probably fail.

**Comment by cc-d** [comment]
Oh man. I'll look into fixing this today. Thanks for the head sup.

**Comment by cc-d** [comment]
So I actually fixed this yesterday. Part of the reason for the difference.

The README is slightly outdated. For my fix I just avoided using getlength all together.


        text_size = (int(text_img.size[0] * 0.75), int(text_img.size[1] * 1))

feel free to open this again if the issue still persits, i'm going to publish 5.1.1 today with some meta info updated too regarding formatting and such.

**Comment by cc-d** [comment]
nevermind i just verified it myself that this is still a problem. still going to look into fixing it today. but this will remain open until then.

**Comment by cc-d** [comment]
fixed as of 5.2.0 let me know if you have any more issues
